gulpfile.js

```diff
@@ -220,7 +220,9 @@ async function generateReactWebviewsBundle() {
 		 * for each entry point, to be used by the webview's HTML content.
 		 */
 		entryPoints: {
-			mssqlwebview: 'src/reactviews/index.tsx'
+			'connectionDialog': 'src/reactviews/pages/ConnectionDialog/index.tsx',
```

Comment:
Adding entry points for each features. New features will have to have their own index.tsx file and we will have to add them here to let esbuild optimally create chunks.